week-2-practise/level-2/5-indexOf.js

```diff
@@ -5,7 +5,8 @@
 // TIP: Use the .indexOf() method
 
 function remove(arr, valueToRemove) {
-  return; // complete this statement
+  return newArr = (arr.slice(0, arr.indexOf(valueToRemove))
```

Comment:
Are you sure this works as expected? Also, it's quite complicated and hard to work out what's happening here - I'd recommend using variables to hold some of this code and then just concatenating the variables at the end